calendar-app/Components/Calendar.js

```diff
@@ -1,45 +1,76 @@
-import React, { useEffect, useState } from 'react';
-import { View, StyleSheet } from 'react-native';
+import React, { useEffect, useState, useCallback } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
 import { Calendar } from 'react-native-calendars';
 import Events from './Events';
 import { getUserId } from './Storage/userDataStorage';
 import { getAllEvents } from './API/Events/AllEventFetcher';
 
-const App = ({ showEvents , onDayPress}) => {
-  const [selected, setSelected] = useState('');
+const CalendarComponent = ({ showEvents, onDayPress, eventMonthCount, getGroupId }) => {
+  const [selectedDate, setSelectedDate] = useState('');
   const [calendarHeight, setCalendarHeight] = useState(350);
   const [events, setEvents] = useState({});
   const [selectedEvents, setSelectedEvents] = useState([]);
+  const [eventCount, setEventCount] = useState(null);
+  const [currentDate, setCurrentDate] = useState(() => {
+    const date = new Date();
+    const year = date.getFullYear();
+    const month = String(date.getMonth() + 1).padStart(2, '0');
+    return `${year}-${month}-01`;
+  });
```

Comment:
It would probably be better to make back end adapt to the current selectedDate format, less code and more understandable

Reply:
Done Boss!
![image](https://github.com/Lukroza/Planner-App/assets/88896783/da6872ea-e9b0-4b13-a69b-28b57d8c694c)
![image](https://github.com/Lukroza/Planner-App/assets/88896783/eed88ad8-ed7a-4c71-9631-bed5a543be13)